### PR TITLE
Guide explaining flashing on OS X Yosemite

### LINKED ISSUE #43: How to use mbed with OS X yosemite
https://developer.mbed.org/users/okano/notebook/mbed-on-yosemite/

### PR DESCRIPTION
**Pull request by stianjensen**
Fixes #43

### CODE REVIEW

File: guides/mbed-yosemite.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+banner: images/mbed-yosemite-banner.jpg
+---
+
+# Flashing your mbed on OS X yosemite
+
+Unfortunately, there is a bug preventing you from flashing your mbed on Yosemite the normal way.
+A fix is on the way, but not released yet.
+
+The easiest way to flash your mbed for now, is therefore using a third-party tool.
+Just download the [mbed on yosemite](assets/mbed-on-yosemite.zip), unzip it and put it in your Applications folder.
```
Comment (sigvef) [inline guides/mbed-yosemite.md]: link is dead

Comment (sigvef) [inline guides/mbed-yosemite.md]: "download the mbed on yosemite app", or "download mbed on yosemite"

Comment (stianjensen) [inline guides/mbed-yosemite.md]: the app is called 'mbed-on-Yosemite' (with dashes, though – is that better?)

Comment (stianjensen) [inline guides/mbed-yosemite.md]: Now I see what you mean, will fix

**Comment by stianjensen** [comment]
plingeling

**Comment by cristeahub** [comment]
I don't understand what I'm suppose to drag my image on to. I think an image or something would help greatly

File: guides/mbed-yosemite.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+layout: guide
+banner: images/mbed-yosemite-banner.jpg
+---
+
+# Flashing your mbed on OS X yosemite
+
+Unfortunately, there is a bug preventing you from flashing your mbed on Yosemite the normal way.
+A fix is on the way, but not released yet.
```
Comment (sigvef) [inline guides/mbed-yosemite.md]: but *is* not released yet

**Pull request merged**